Makefile

```diff
@@ -87,14 +87,14 @@ gencatalog:
 	rm site/content/en/guides/consumer/function/catalog/*/_index.md
 	(cd site/content/en/guides/consumer/function/catalog/catalog && npm run gen-docs)
 
-run-site:
-	(cd site && python -m SimpleHTTPServer 3000)
+site-run-server:
+	npx http-server site/ -p 3001
 
-check-site:
+site-check:
 	./scripts/checkSite.sh
```

Comment:
nit: bash scripts usually use name convention `check-site` or `check_site` but not `checkSite`.